ChangeLog

```diff
@@ -422,7 +422,8 @@ and bug requests.
    * filter.d/uwimap-auth - filter for uwimap-auth IMAP/POP server
   Steven Hiscocks and Daniel Black
    * filter.d/selinux-{common,ssh} -- add SELinux date and ssh filter
-
+  Donald Yandt
+   * action.d/firewallcmd-{multiport,allports}.conf --added two new actions
```

Comment:
I am puzzled -- why are you changing changelog for the very old release? ;)  modify the one on top of the file for 0.9.2, not in the middle ;-)